spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -67,29 +68,24 @@ def verify_session_cookie
       context 'routes not requiring auth' do
         %w[mhv dslogon idme].each do |type|
           context "routes /sessions/#{type}/new to SessionsController#new with type: #{type}" do
-            it 'returns JSON' do
-              get(:new, type: type, format: :json)
-              expect(response).to have_http_status(:ok)
-              expect(JSON.parse(response.body).keys).to eq %w[url]
-            end
-
             it 'redirects' do
               get(:new, type: type)
               expect(response).to have_http_status(:found)
+              expect(response.location)
+                .to be_an_idme_saml_url('https://api.idmelabs.com/saml/SingleSignOnService?SAMLRequest=')
+                .with_relay_state('originating_request_id' => nil)
```

Comment:
@narin for these specs, you can use just add a .with_param and test out the clientID is getting set but you'll have to setup the request to receive that param as well, so you change get(:new, type: type) to be get(:new, type: type, ga_id: '12312313') and .with_param('ga_id' => '12312313') for all of the new actions. You'll not that there are two sections where these are tested (once for when you are signed in and once for when you are not).